front/src/modules/ui/right-drawer/components/RightDrawer.tsx

```diff
@@ -56,7 +56,7 @@ export const RightDrawer = () => {
   useListenClickOutside({
     refs: [rightDrawerRef],
     callback: () => closeRightDrawer(),
-    mode: ClickOutsideMode.absolute,
+    mode: ClickOutsideMode.dom,
```

Comment:
Changing this to dom because the drawer was closing when I clicked on upload image